characteristic_notify/index.html

```diff
@@ -62,14 +62,16 @@ <h2>Step 2</h2>
     <p>Status: <span id='test_result'></span></p>
     <pre id='status'></pre>
   </div>
-  <!-- Only one of the two below will be visible. -->
-  <div id="no_bluetooth" class="no-bluetooth" style="visibility: hidden;">
-    Bluetooth is not available.
+  <!-- Only one of the three below will be visible. -->
+  <div id="bluetooth_none" class="no-bluetooth" style="visibility: hidden;">
+    Bluetooth is not present.
```

Comment:
To be more specific I would say "Web Bluetooth is not supported by this browser."